plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4402,6 +4402,16 @@ public void testSelectWithMoreThanOneSnapshotOfTheSameTable()
         assertUpdate(format("DROP TABLE %s", tableName));
     }
 
+    @Test
+    public void testInsertingIntoTablesWithColumnsWithQuotesInName()
+    {
+        String tableName = "tesT_inserting_into_tables_with_quotes_" + randomTableSuffix();
```

Comment:
I assume the uppercase `T` is not intentional, and it distracts from the purpose of the test.